packages/core/src/components/HelpDrawer/HelpDrawer.scss

```diff
@@ -35,20 +35,26 @@
 }
 
 .ds-c-help-drawer__body {
-  flex-grow: 1; // Stretch so that the container takes up the rest of the screen
-  position: relative;
-  width: 100%;
+  // flex-grow: 1; // Stretch so that the container takes up the rest of the screen
+  // position: relative;
+  // width: 100%;
+    // height: 100vh;
+    overflow-y: scroll;
 }
 
 .ds-c-help-drawer__content-container {
-  bottom: 0;
+  // bottom: 0;
   display: flex; // Flex layout helps stick the footer to the bottom
   flex-direction: column;
-  left: 0;
-  overflow: scroll;
-  position: absolute;
-  right: 0;
-  top: 0;
+  height: calc(100vh - 60px);
```

Comment:
don't feel great about this height calculation and am wondering if there is a better way

Reply:
what is the height calculating? the footer height?

Reply:
I'm getting this: https://gyazo.com/29616b924a6aebd5bdc752748b0401ba